src/components/EditorCanvas/Table.jsx

```diff
@@ -266,14 +266,18 @@ export default function Table(props) {
             ? ""
             : "border-b border-gray-400"
         } group h-[36px] px-2 py-1 flex justify-between items-center gap-1 w-full overflow-hidden`}
-        onMouseEnter={() => {
+        onPointerEnter={(e) => {
```

Comment:
Seems like `onPointerEnter` does not trigger in the same way when using non-mouse pointers

Reply:
Fixed